docs/external-content/application-connector-manager/docs/user/_sidebar.ts

```diff
@@ -0,0 +1,32 @@
+export default [
```

Comment:
Same

Will this be hardcoded? If yes, then it would be good to configure it prettier because now it's really hard to read. If not, then it should be in .gitignore.

Reply:
same as the precedent comment